src/components/Login/Login.tsx

```diff
@@ -25,19 +25,7 @@ export function Login() {
 			return;
 		}
 
-		loginContext.logIn(authHeader);
-		setError(undefined);
-
-		try {
-			await fetchIsAdmin(authHeader);
-		}
-		catch {
-			console.log("You're logged in as a member");
-			return;
-		}
-
-		loginContext.setUserAdmin(true);
-		console.log("You're an admin");
+		loginContext.logIn(authHeader, await fetchIsAdmin(authHeader));
```

Comment:
Can we change this to set the fetch response to a variable and pass it in?
Just makes it slightly easier to read
```suggestion
                const isAdmin = await fetchIsAdmin(authHeader)
		loginContext.logIn(authHeader, isAdmin);
```